cibuildwheel/options.py

```diff
@@ -672,6 +682,9 @@ def build_options(self, identifier: str | None) -> BuildOptions:
                 "test-requires", option_format=ListFormat(sep=" ")
             ).split()
             test_extras = self.reader.get("test-extras", option_format=ListFormat(sep=","))
+            test_groups_str = self.reader.get("test-groups", option_format=ListFormat(sep=","))
+            test_groups = [x for x in test_groups_str.split(",") if x]
+            test_dependency_groups = self.dependency_groups(*test_groups)
```

Comment:
The naming of this  `test_dependency_groups` variable is a little confusing. These aren't groups anymore, they've been resolved into package requirements.

My suggestion would be `requirements_from_test_groups`.

Reply:
I went with `test_requirements_from_groups`.